docker/entrypoint/docker-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+set -e
+
+APP_DIR="/opt/tarantool"
+DATA_DIR="/var/lib/tarantool/sys_env"
+LOG_DIR="/var/log/tarantool/sys_env"
+RUN_DIR="/var/run/tarantool/sys_env"
```

Comment:
Shouldn't it be `/run/tarantool/sys_env` (and, similarly, `mkdir -p /run/tarantool` in the Dockerfile)?

https://lwn.net/Articles/436012/
https://wiki.debian.org/ReleaseGoals/RunDirectory
https://wiki.ubuntu.com/OneiricOcelot/ReleaseNotes#Upgrades
https://bugs.gentoo.org/361349

It seems, it was changed from `/var/run` to `/run` in all popular Linux distros around 2011.

Reply:
Actually, it doesn't matter, `/var/run` or `/run`. `/var/run` is a symlink to `/run`. By the way, `tt` also uses `/var/run`.

Reply:
I think that 13 years after `/run` became a primary directory we can use it instead of the symlink. But OK, AFAIK using `/var/run` doesn't lead to any user visible problems.